src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -412,11 +412,11 @@ public override void OnInitialize()
         /// Informs AT-SPI bridge about the set of AT-SPI states associated with this object.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override AccessibilityStates AccessibilityCalculateStates()
+        protected override AccessibilityStates AccessibilityCalculateStates(ulong states)
         {
-            var states = base.AccessibilityCalculateStates();
-            FlagSetter(ref states, AccessibilityStates.Modal, true);
-            return states;
+            var _states = base.AccessibilityCalculateStates(states);
```

Comment:
if possible, we are recommended not to use `_`. 
could you remove `_` ?

Reply:
Thank you for your review.
Updated.